compass/landice/tests/ismip6_run_ais/set_up_experiment.py

```diff
@@ -0,0 +1,215 @@
+import os
+from compass.model import run_model, make_graph_file
+from compass.step import Step
+import shutil, glob, sys
+from jinja2 import Template
+from importlib import resources
+
+
+class SetUpExperiment(Step):
+    """
+    A step for setting up an ISMIP6 experiment
+
+    Attributes
+    ----------
+    """
+
+    def __init__(self, test_case, name, subdir, exp):
+        """
+        Set up a new experiment
+
+        Parameters
+        ----------
+
+        exp : experiment
+        """
+
+        self.exp = exp
+
+        super().__init__(test_case=test_case, name=name, subdir=subdir)
+
+    def setup(self):
+
+        print(f"    Setting up experiment {self.exp}")
+
+        if self.exp == 'hist':
+            exp_fcg = 'ctrlAE'
+        else:
+            exp_fcg = self.exp
+
+        config = self.config
+        section = config['ismip6_run_ais']
+        mesh_res = section.get('mesh_res')
```

Comment:
I would get this as an integer and add the zero padding yourself.
```suggestion
        mesh_res = section.getint('mesh_res')
```